src/components/ui/Roster/RosterGroup.tsx

```diff
@@ -5,18 +5,18 @@ import React from 'react';
 
 import Badge from '../Badge';
 import Flex from '../Flex';
-import { BaseProps } from '../Base';
+import { BaseProps, FocusableProps } from '../Base';
 
 import { StyledGroupWrapper, StyledGroup, StyledTitle } from './Styled';
 
-interface Props extends BaseProps {
+export interface RosterGroupProps extends BaseProps, FocusableProps {
```

Comment:
FWIW, I don't think I need this. but it doesn't hurt anything to have it.

Reply:
I'll just keep in the chance that it's needed later - like you said, shouldn't hurt